control-plane/agents/src/bin/core/tests/volume/capacity_limit.rs

```diff
@@ -0,0 +1,88 @@
+#![cfg(test)]
+
+use deployer_cluster::ClusterBuilder;
+use grpc::operations::volume::traits::VolumeOperations;
+use std::time::Duration;
+use stor_port::{
+    transport_api::ReplyErrorKind,
+    types::v0::transport::{CreateVolume, DestroyVolume},
+};
+use uuid::Uuid;
+
+const SIZE: u64 = 5242880;
+const EXCESS: u64 = 1000000;
+
+#[tokio::test]
+async fn volume_create_with_capacity_limit() {
+    let cache_period = Duration::from_millis(250);
+    let reconcile_period = Duration::from_millis(3000);
+    let cluster = ClusterBuilder::builder()
+        .with_rest(true)
+        .with_io_engines(1)
+        .with_pool(0, "malloc:///p1?size_mb=300")
+        .with_cache_period(&humantime::Duration::from(cache_period).to_string())
```

Comment:
Doesn't this directly work?
```suggestion
        .with_cache_period(&cache_period.to_string())
```

Reply:
Apparently not: `std::time::Duration` doesn't implement `std::fmt::Display`
It is also constructed this way in the other tests.